magento/ece-tools
# MAGECLOUD-3475: Refactor Shell implementation

### Description

The current implementation of Shell does not expect to have non-zero output from the command. This affects few commands which can return different codes like setup:db:status.

In this PR changed the implementation of Shell command to return a result with output message and exit code.



### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3475

### Manual testing scenarios

1. Deploy Magento to cloud.
2. Check that all shell commands are running as usual.

https://jira.corp.magento.com/browse/MAGECLOUD-117

### Contribution checklist
